kube/services/revproxy/00nginx-config.yaml

```diff
@@ -78,13 +78,30 @@ data:
           location /user {
               proxy_pass http://fence-service/;
           }
+          location /api/v0/submission/graphql {
```

Comment:
should direct `/api/v0/submission/getschema` to [peregrine](https://github.com/uc-cdis/peregrine/blob/fca3b33b76ef4b900f08b27b4678a11f15fbc9c8/peregrine/resources/submission/__init__.py#L103) too